chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+namespaces="${NAMESPACES}"
+timeout="${TIMEOUT}"
+ignoreTimeoutError="${IGNORETIMEOUTERROR}"
+
+if [ -z "${namespaces}" ]; then
+  echo "No namespace is provided."
+  exit 1
+fi
+
+if [ -z "${timeout}" ]; then
+  echo "No timeout value is provided."
+  exit 1
+fi
+
+if [ -z "${ignoreTimeoutError}" ]; then
+  echo "No ignoreTimeoutError value is provided."
+  exit 1
+fi
+
+succeeded=()
+failed=()
+
+get_pod_count() {
+  kubectl get pods --selector app="${1}" -n "${2}" -o json | jq '.items | length'
+}
+
+echo "Uninstalling Rancher resources in the following namespaces: ${namespaces}"
+
+for namespace in ${namespaces}; do
+  for app in $(helm list -n "${namespace}" -q); do
+    if [[ ${app} =~ .crd$ ]]; then
```

Comment:
```suggestion
    if [[ "${app}" =~ ".crd$" ]]; then
```

Reply:
The double brackets `[[` prevents the word splitting by default, so the quotes around the string variable are optional. 
I replaced other `[`s with `[[`s in the file and no quotes to make use of the good feature of  `[[`.